setup.py

```diff
@@ -19,9 +21,21 @@
       url='https://github.com/AtteqCom/zsl',
       license='MIT',
       install_requires=[
-        'flask_injector==0.8.0',
-        'future'
+          'future',
+          'Flask-Injector>=0.8' if _is_py3 else 'Flask-Injector==0.8.0',
```

Comment:
For now I would fix it to 0.8 in setup.py, I think we don't use anything special from 0.9.